src/base/memory_monitor_sr.cpp

```diff
@@ -73,6 +69,18 @@ namespace cubmem
     std::string target ("/src/");
 #endif // !WINDOWS
 
+    // TODO: To minimize the search cost and prevent unnecessary paths
+    //       from being included in the tag_name, we are cutting the string based on
+    //       the rightmost "/src/" found. However, in this case, when the allocation
+    //       occurs on the same 'line', "/src/test.c" and "/src/thirdparty/src/test.c"
+    //       get the same tag_name, making it impossible to distinguish memory alloc-
+    //       ations from two different files. However, such exceptional situations
+    //       are very rare, as memory allocations must occur on the same line number.
+    //       Handling these exceptions would increase the overall cost of this function.
+    //       Moreover, currently, although these exceptional situations theoretically exist,
+    //       they have not been encountered in developer tests.
+    //       Therefore, it has been decided to address them if discovered in the future.
```

Comment:
Hmm, since this is an implementation part that leaves room for potential malfunctions rather than optimization, I think it would be better to create issue as a backlog to track easily rather than leaving it as a TODO comment. What do you think?

Reply:
In fact, I'm worried and skeptical.

1. Will we be able to recognize when a problem occurs?
2. if there's a potential issue in the future, when will it be addressed?
3. can the person creating a new directory be aware of this matter?

When it comes to maintenance, we often forget most of the context. 
Even if we don't create an issue, it would be good to write constraints in the epic

Reply:
I added it  as limitation part in http://jira.cubrid.org/browse/CBRD-25201.